src/character/character-sheet/edit/EditResources.tsx

```diff
@@ -105,62 +156,132 @@ export const EditResources = () => {
+              onResourceSegmentClick={updateFatigue}
+            />
+          </ButtonCounterWrapper>
+        </ResourceSegment>
+        {showPowerPoints ? (
+          <ResourceSegment>
+            Power Points:
+            <ButtonCounterWrapper>
+              <Button
+                text="X"
+                customcolor={Theme.PowerPoints}
+                onClick={() => updatePowerPoints(0)}
+              />
+              <ResourceCounter
+                total={maxPowerPoints}
+                remaining={currentPowerPoints}
+                color={Theme.PowerPoints}
+                onResourceSegmentClick={updatePowerPoints}
+              />
+            </ButtonCounterWrapper>
+          </ResourceSegment>
+        ) : null}
+        <IconButton
+          icon={Icons.Campfire}
+          viewbox="0 0 1080 1296"
+          onClick={fullRest}
+          text="Full Rest"
+        />
+        <ResourceSegment>
+          <FlexRow>
+            <TextElement>Temporary Effects</TextElement>
+            <IconButton
+              onClick={() => setShowAddBuffModal(true)}
+              icon={Icons.Add}
+              text={"Add"}
+            />
+          </FlexRow>
+          {Boolean(temporaryEffects) ? (
+            <TemporaryEffectsTable>
+              <thead>
+                <tr>
+                  <th>Name</th>
+                  <th>Duration</th>
+                </tr>
+              </thead>
+              {dataObjectToList(temporaryEffects!).map((temporaryEffect) => (
+                <tr key={temporaryEffect.name + temporaryEffect.duration}>
+                  <td className="item-name">
+                    <TextElement title={temporaryEffect.name}>
+                      {temporaryEffect.name}
+                    </TextElement>
+                  </td>
+                  <td className="item-duration">
+                    <DurationRowControls>
+                      <TextElement title={temporaryEffect.name}>
+                        {temporaryEffect.duration}
+                      </TextElement>
+                      <IconButton
+                        iconSize={16}
+                        icon={Icons.ChevronDown}
+                        onClick={() =>
+                          tickTemporaryEffectDuration(
+                            character.id,
+                            temporaryEffect,
+                            true
+                          )
+                        }
+                      />
+                      <IconButton
+                        iconSize={16}
+                        icon={Icons.ChevronUp}
+                        onClick={() =>
+                          tickTemporaryEffectDuration(
+                            character.id,
+                            temporaryEffect,
+                            false
+                          )
+                        }
+                      />
+                    </DurationRowControls>
+                  </td>
+                </tr>
```

Comment:
Table elements are not supposed to be children of anything else than `<table>` according to `validateDOMNesting` rule. I assume the intent was to make `TemporaryEffectsTable` a `styled.table` instead of `styled.ul`.